src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -574,23 +620,10 @@ where
         config: &MainGateConfig<T>,
         mut ro_circuit: impl ROCircuitTrait<C::Base>,
         public_params_hash: &C::Base,
-        instance: &PlonkInstance<C>,
+        input_plonk: &PlonkInstance<C>,
```

Comment:
NBD: `instance` is better

Reply:
Considering how many times I've been confused between `instance` & `instance.instance` I'd say this title is a bit confusing